_data/simple-icons.json

```
@@ -10929,8 +10929,9 @@
         },
         {
             "title": "OSGeo",
-            "hex": "5CAE58",
-            "source": "https://www.osgeo.org"
+            "hex": "4CB05B",
+            "source": "https://github.com/OSGeo/osgeo/blob/5ee0625fbef339d6badc14cdd121d0174161a192/marketing/branding/logo/osgeo-logo-bw.svg",
+            "guidelines": "https://github.com/OSGeo/osgeo/blob/5e3df1231cfd1f3a1cb087dd541a4f9b8ed85a89/marketing/branding/styleguide-osgeo.pdf"
```

For `guidelines` on GitHub, we use the default branch rather than the commit hash to ensure that we're always providing the latest version.
```suggestion
            "guidelines": "https://github.com/OSGeo/osgeo/blob/master/marketing/branding/styleguide-osgeo.pdf"
```

---

I noticed a lot of other guidelines use commit hashes for stability/reproducibility. Is this a new policy? I'm fine with whatever you decide.

---

Ah, it looks like source is required to use the hash but guidelines isn't. Would be good to make these consistent and add tests to enforce it.